docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -252,12 +255,11 @@ Here's a breakdown of what/why:
 
 ### Use kubeadm with other CRI runtimes
 
-Since the [Kubernetes 1.6 release](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md#node-components-1),
-Kubernetes has started using CRI as the default container runtime.
-The container runtime used by kubeadm is Docker, which is enabled through the built-in
-`dockershim` in `kubelet` mechanisms.
+Since v1.6.0, Kubernetes has enabled the use of CRI, Container Runtime Interface, by default.
+The container runtime used by default is Docker, which is enabled through the built-in
+`dockershim` CRI implementation inside of the `kubelet`.
 
-Other CRI-based runtimes include::
+Well-known, external CRI-based runtimes include:
```

Comment:
I liked the previous statement with a single `:`

Reply:
@luxas ☝️

Reply:
fixed